packages/grid/_modules_/grid/hooks/features/useComponents.tsx

```diff
@@ -59,6 +57,7 @@ export const useComponents = (
     };
 
     return {
+      ...allComponents,
```

Comment:
I think that we have a similar problem to https://github.com/mui-org/material-ui-x/pull/851#discussion_r559815532 (that we didn't resolve yet). There is no fallback: see bug 2 in https://codesandbox.io/s/material-demo-forked-m7ilt?file=/demo.js (you need to resize)